oshi-core/src/main/java/oshi/SystemInfo.java

```diff
@@ -47,9 +48,7 @@
  * platform-specific implementations of {@link oshi.software.os.OperatingSystem}
  * (software) and {@link oshi.hardware.HardwareAbstractionLayer} (hardware).
  */
-public class SystemInfo implements Serializable {
```

Comment:
Good call... I'll mention it but don't think it requires a major version bump.

Reply:
I happen to disagree with this, I simply can't see how implementing this "marker interface" can harm anybody. It's like if the whole world was only about web services, and the only thing that "matters" is the risk that follows by deserializing untrusted or otherwise "manipulatable" sources. It is however entirely up to the application developer if he/she will use serialization, and what sources that will be used for deserialization. 

I don't mean to say that serialization is "perfect" in any way, I wish it had more capabilities, especially that it was possible to initialize final fields from a custom deserializer. That said, it's certainly better than nothing, or having to include huge external libraries to achieve serialization if the needs are simple. I don't see the big problem with using it for things like caching, either in a local database or in memory, for many applications.

Why take away these possibilities just because some web developers abuse the functionality..?